common/pom.xml

```diff
@@ -78,6 +78,12 @@
       <artifactId>log4j-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.reactor.blockhound</groupId>
+      <artifactId>blockhound</artifactId>
+      <version>30c9eb9f2c</version>
```

Comment:
Refers to https://github.com/reactor/BlockHound/commit/30c9eb9f2c, will remove once 1.0.1 is released

Reply:
k... That said I think before we merge we need to wait for a real release.

Reply:
of course! Especially given that it is an optional dependency 👍

Reply:
It will hold the release up to the point where everything works in Netty and expect to release it today or tomorrow (EU time)